src/test/java/de/qabel/desktop/ui/accounting/AccountingControllerTest.java

```diff
@@ -67,39 +70,37 @@ public void addsIdentitiesWithAlias() throws Exception {
 	}
 
 	@Test
-	public void exportTest() throws IOException, QblStorageException, URISyntaxException {
-		File testDir = setupImportAndExport();
-		identity = new Identity(TEST_ALIAS, null, null);
-		controller.exportIdentity(identity, testDir);
-		File f = new File(TEST_FOLDER + "/" + TEST_ALIAS + ".json");
-		assertEquals(f.getName(), TEST_ALIAS+ ".json");
+	public void exportIdentityTest() throws IOException, QblStorageException, URISyntaxException {
```

Comment:
This test tests nothing. You can delete `controller.exportIdentity` without impact.